imports/lib/models/settings.js

```diff
@@ -1,18 +0,0 @@
-import { Roles } from 'meteor/nicolaslopezj:roles';
-import Base from './base';
-import SettingsSchema from '../schemas/settings';
-
-const Settings = new Base('settings');
-Settings.attachSchema(SettingsSchema);
-
-function queryModifier(q) {
-  // Only allow admins to pull down Settings.
-  if (Roles.userHasRole(this.userId, 'admin')) {
-    return q;
-  }
-
-  // Make the query evaluate to nothing
-  return { $and: [false, q] };
```

Comment:
Technically this was invalid, since the list elements for `$and` are supposed to be query hashes, not booleans. I think if this query had actually run, it would have resulted in an error (which is maybe close enough to desired)